kevm-pyk/src/kevm_pyk/kevm.py

```diff
@@ -344,6 +344,10 @@ def abi_bool(b: KInner) -> KApply:
     def abi_type(type: str, value: KInner) -> KApply:
         return KApply('abi_type_' + type, [value])
 
+    @staticmethod
+    def abi_tuple(values: Sequence[KInner]) -> KApply:
+        return KApply('abi_type_tuple', values)
```

Comment:
For generality and matchnig the signature of `KApply` (and other methods here), it should probably use `Iterable`:

```suggestion
    @staticmethod
    def abi_tuple(values: Iterable[KInner]) -> KApply:
        return KApply('abi_type_tuple', values)
```

https://github.com/runtimeverification/pyk/blob/79c45d9f1ae355b591cbd2e8b77c6205ab170835/src/pyk/kast/inner.py#L368

Reply:
Thanks! Fixed in https://github.com/runtimeverification/evm-semantics/pull/2105/commits/06b2b44d379d6a57878aad3f8d23a30ce390cd4a.